### PR TITLE
Feature/131 invoice ants

### PR DESCRIPTION
**Pull request by AlexAnarcho**
Invoice as a Modal, no tailwindcss anymore, however functionality with the actual payment/confirmation is still missing. Created a new Ticket (133) for this task.

Modal can be viewed currently at localhost.../invoice

Also created helper classes in styles/index.less for text-center and text-left

### CODE REVIEW

File: src/pages/InvoiceModal.jsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    </div>
+  );
+};
+
+const Payment = () => {
+  const [yearly, setYearly] = useState(true);
+  const [isPremium, setIsPremium] = useState(true);
+  const [amount, setAmount] = useState(0);
+  const [qrCode, setQrCode] = useState("");
+  const [paymentUri, setPaymentUri] = useState(null);
```
Comment (hundehausen) [inline src/pages/InvoiceModal.jsx:61]: remove because unnessesary

File: src/pages/InvoiceModal.jsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+  const createPaymentUri = () => {
+    let uri;
+    if (amount > 0) {
+      uri = "monero:" + subaddress + "?tx_amount=" + amount;
+    } else {
+      uri = "monero:" + subaddress;
+    }
+    setPaymentUri(uri);
+    return uri;
```
Comment (hundehausen) [inline src/pages/InvoiceModal.jsx:95]: const createPaymentUri = () => {
   return amount > 0 ? `monero:${subaddress}?tx_amount=${amount}` : `monero:${subaddress}`;
}

File: src/pages/InvoiceModal.jsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  };
+
+  // generete QR Code on subaddress change
+  useEffect(() => {
+    const paymentUri = createPaymentUri();
+    const generateQrCode = async () => {
+      if (subaddress !== null) {
```
Comment (hundehausen) [inline src/pages/InvoiceModal.jsx:102]: if (subaddress)

**Review by hundehausen — CHANGES_REQUESTED** [review]
I know the code I annotated is from me 🤪

File: src/pages/InvoiceModal.jsx (hunk 1 above)
Comment (AlexAnarcho) [reply src/pages/InvoiceModal.jsx:61]: I dont understand. these states are necessary for the payment.

**Review by hundehausen — APPROVED** [review]

**Pull request merged**